### PR TITLE
[sc-2398] Fix Image blocks stripping out links

### PR DESCRIPTION
**Pull request by SpaYco**
enabling all attributes on all elements is what's causing this problem, not sure exactly why, but to fix this, in the `upcastImageFigure`, i'm using the anchor when available instead of the image, so it doesn't strip out the link, from my tests, ~everything worked fine.~

known issues:

- if you enable the extra options (eg: open in new tab) and refresh the page, the button will be shown disabled even though in the code view, the options are there, I'm still looking into this, but it's probably unrelated and needs to be fixed in a separate PR.

fixes [sc-2398]

### CODE REVIEW

File: packages/ckeditor5-build-helpjuice-v2/build/translations/it.js
Comment (NicholasJarr) [inline packages/ckeditor5-build-helpjuice-v2/build/translations/it.js:1]: No need for this file to be committed, right?

**Review by NicholasJarr — APPROVED** [review]
It worked without issues for me (except the one you explained that will be handled in a separate PR).

I also left a question

**Comment by NicholasJarr** [comment]
@SpaYco

You should also merge this PR now that the PR in the repo was also merged.

**Comment by SpaYco** [comment]
@NicholasJarr sorry, somehow i missed this one, thanks for the reminder!!

**Pull request merged**